code-samples/creating-your-bot/command-handling/deploy-commands.js

```diff
@@ -13,15 +13,6 @@ for (const file of commandFiles) {
 
 const rest = new REST({ version: '9' }).setToken(token);
 
-(async () => {
-	try {
-		await rest.put(
-			Routes.applicationGuildCommands(clientId, guildId),
-			{ body: commands },
-		);
-
-		console.log('Successfully registered application commands.');
-	} catch (error) {
-		console.error(error);
-	}
-})();
+rest.put(Routes.applicationGuildCommands(clientId, guildId), { body: commands })
+	.then(() => { console.log('Successfully registered application commands.'); })
```

Comment:
```suggestion
	.then(() => console.log('Successfully registered application commands.'))
```
Since the return value doesn't matter here, this can be shortened